recipes/tinyfasta/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "tinyfasta" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "ccbb388eeb1809260be1e4893ebff0dd5b25d18e1a06b7c328e5803e8e924287" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tjelvar-olsson/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
I guess we should use this now: https://conda-forge.org/docs/meta.html#use-pip

Reply:
Hmmm.... not sure how that broke the build...